src/main/java/com/anywhereworks/bookmark/globalExceptionHandler/Exceptions.java

```diff
@@ -0,0 +1,39 @@
+package com.anywhereworks.bookmark.globalExceptionHandler;
+
+import jakarta.persistence.EntityNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Exceptions {
```

Comment:
```suggestion
@RestControllerAdvice
public class GlobalExceptionHandler {
```
would be better here

Reply:
Added changes